c/busybox-1.22.0/busybox_sv_comp-getpwnam.h

```diff
@@ -0,0 +1,28 @@
+#define getpwnam bb_internal_getpwnam
+
+struct passwd *getpwnam(const char *name)
+{
+  (void)name;
+
+  static struct passwd p;
+
+  p.pw_name = "";       /* username */
+  p.pw_passwd = "";     /* user password */
```

Comment:
The benchmarks use these values only for printing, so I guess the empty strings are OK.